git/operations.go

```diff
@@ -250,13 +267,13 @@ func changed(ctx context.Context, path, ref string, subPaths []string) ([]string
 		args = append(args, subPaths...)
 	}
 
-	if err := execGitCmd(ctx, path, out, args...); err != nil {
+	if err := execGitCmd(ctx, path, out, nil, args...); err != nil {
 		return nil, err
 	}
 	return splitList(out.String()), nil
 }
 
-func execGitCmd(ctx context.Context, dir string, out io.Writer, args ...string) error {
+func execGitCmd(ctx context.Context, dir string, out io.Writer, extraEnv []string, args ...string) error {
```

Comment:
Since the `extraEnv` parameter is only used exceptionally, I would  propose to split the function in two: `execGitCmd` (without env variables) and `execGitCmdWithEnvs`

Alternatively, since the amount of arguments is running out of hand, we could use a struct and omit the `extraEnv` field when not needed (since it would default to nil).

Reply:
Did the latter.